src/main/java/Interfaces/InputOutputInterfaces.java

```diff
@@ -3,6 +3,7 @@
 import java.io.IOException;
 import java.net.Socket;
 
+
```

Comment:
Check this out: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat-on-save You can set up IntelliJ to automatically reformat code when you save.

Try giving that a once-over and then commit that on this branch. It'll make the diffs a little easier to read.